src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import './Article.css';
+
+
+export const ArticleCard = ({ article }) => {
```

Comment:
Good to see you using an `ArticleCard` and passing an individual article through as props, well done